src/site/content/en/blog/quintoandar/index.md

```diff
@@ -0,0 +1,226 @@
+
+The framework provides other performance-friendly features such as automatic [code splitting](/reduce-javascript-payloads-with-code-splitting/) and [prefetching](/link-prefetch/). Even though the existing structure already provided such features, the additional work required from developers stalled their adoption — for example, code splitting at page or component-level had to be done manually.
+
+### Optimizing JavaScript resources
+
+The first step was to [remove unused code](/remove-unused-code/). We looked at the [Webpack Bundle Analyzer](https://github.com/webpack-contrib/webpack-bundle-analyzer) reports, which shows the contents of each JS bundle, and carefully reviewed all third-party scripts. As a result, we were able to clean up some tracking libraries that were not used in this specific page.
+
+Our team went further and evaluated the performance cost of existing features. For instance, the "like" button required quite a lot of JS to work. However, in the condominium page, less than 0.5% of the users interacted with the button, which is available and used more frequently in other parts of our app. After a discussion involving both Engineering and Product, we decided to remove this feature.
+
+<figure>
+  {% Video src="video/jL3OLOhcWUQDnR4XjewLBx4e3PC3/2uEL2YxbuGGlmJDlvW9F.mp4", autoplay=true, muted=true, playsinline=true, loop=true, controls=true %}
+  <figcaption>
+    An animation showing the “like” button feature. There is a card about an apartment available for rent. In the bottom right corner of the card, there is a grey heart-shaped button that turns blue when clicked.
+  </figcaption>
+</figure>
+
+Other JS optimizations were already in place, such as [static compression with Brotli](/reduce-network-payloads-using-text-compression/#static-compression), which was done at build time using [`BrotliWebpackPlugin`](https://github.com/mynameiswhm/brotli-webpack-plugin), and was also applied to other types of static resources. At first, we were relying on the compression provided by the CDN, and Brotli reduced JS size by 18% compared to gzip. But then, we switched to Brotli compression at build time, and were able to achieve a 24% reduction.
+
+### Optimizing image resources
+
+There is a hero image occupying most of the area above the fold in the mobile version. It also happens to be the [Largest Contentful Paint](//lcp/) (LCP) of the page.
+
+<figure>
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/W7koRinBm3QXErIzwGP8.png", alt="The condominium page for Edifício Copan (São Paulo, Brazil). A photo taken from the ground level shows the curves of the building structure.", width="800", height="1422" %}
```

Comment:
I think this image will look better scaled down:

```suggestion
  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/W7koRinBm3QXErIzwGP8.png", alt="The condominium page for Edifício Copan (São Paulo, Brazil). A photo taken from the ground level shows the curves of the building structure.", width="360", height="640" %}
```

Also, are we able to center this image?